### PR TITLE
[RPL2-130] Mapping for Ripple Tide Landing Page API

### PR DESCRIPTION
**Pull request by dylankelly**
## Motivation and Context

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

## Changed

1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

File: examples/reference/nuxt.config.ts [hunk 1]
```diff
@@ -11,33 +11,12 @@ const config: NuxtConfig = {
   },
   publicRuntimeConfig: {
     SITEID
-  },
-    proxy: {
```
Comment (dylankelly) [inline examples/reference/nuxt.config.ts:15]: moved to Tide API module

File: examples/reference/pages/_.vue [hunk 2]
```diff
@@ -1,38 +1,19 @@
 <template>
-  <div class="rpl-page" v-if="page && !page.error">
-    <component :is="pageComponent" :page="page"></component>
-  </div>
-  <error-page v-else-if="error" :type="error"></error-page>
+  <rpl-page :page="page">
```
Comment (dylankelly) [inline examples/reference/pages/_.vue:2]: `rpl-page` wraps all page template logic

File: packages/ripple-nuxt/components/ripple-page.vue [hunk 3]
```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="rpl-page" v-if="page && !page.error" :class="page.class">
+    <rpl-page-layout
+      :sidebar="hasSidebar || Boolean($slots.sidebar)"
+      :backgroundColor="page.backgroundColor"
+      :backgroundGraphic="heroBgGraphic"
+      class="app-main"
+    >
+      <slot name="banner">
```
Comment (dylankelly) [inline packages/ripple-nuxt/components/ripple-page.vue:9]: Adds slots so content can be overridden in client projects

File: packages/ripple-tide-api/src/services/utils/default-mapping.js [hunk 4]
```diff
@@ -6,5 +6,13 @@ export default {
     type: data => data.type && data.type.replace('node--', ''),
     nid: 'drupal_internal__nid'
   },
-  includes: []
+  includes: [
```
Comment (dylankelly) [inline packages/ripple-tide-api/src/services/utils/default-mapping.js:9]: These are default includes lifted from ripple-nuxt-tide https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/lib/config/tide.config.js#L5

File: packages/ripple-tide-api/src/services/utils/index.js [hunk 5]
```diff
@@ -47,6 +47,17 @@ export const getLinkFromField = (field, path) => {
   return { text: url && text === '' ? url : text, url }
 }
 
+export const getAddress = (field) => {
+  // TODO: refactor this please for readability
```
Comment (dylankelly) [inline packages/ripple-tide-api/src/services/utils/index.js:51]: I haven't got this working yet, but can follow up in another PR

**Review by ChrisCampbellCodes — APPROVED** [review]

**Pull request merged**